### ISSUE TITLE
Client Support?

### ISSUE THREAD

**Issue by rkjnsn** [pfpacket/rust-9p#3]
Would you have any interest in incorporating client support into the library?

**Comment by pfpacket** [comment]
No. There aren't any plans at all to implement the 9P client support.
It's because rust-9p is intended to be used on Linux systems (at least for now) and Linux kernel has its native 9p client support  (called v9fs) in the form of a kernel module.
If you're using BSD systems you might want to use FUSE 9p clients.
May I ask why you need the client support?

**Comment by rkjnsn** [comment]
I was specifically interested in writing a simple FUSE client for 9P2000.L, since it looks like the existing couple I found only support 9P2000 and/or 9P2000.u. Specifically, I'd like to access a Diod server from non-Linux systems. If there is an existing solution for this, I'd appreciate if you could point me in the right direction.

Thanks!
